File: find_multiples_of_a_number/fmoan.java
// [find multiples of a number]
// [8 kyu]
// [java]
public class Multiples
{
public static int[] find(int b, int l)
{
int[] ns = new int [(int) l / b];
for (int i = 0, n = b; n <= l; i++, n += b)
{
ns[i] = n;
}
return ns;
}
}

File: find_multiples_of_a_number/fmoan.lua
-- [find multiples of a number]
-- [8 kyu]
-- [lua]
local solution = {}
function solution.find_multiples(integer, limit)
local multiples = {} 
for i = integer, limit, integer do
table.insert(multiples, i)
end
return multiples
end 
return solution